demisto_sdk/commands/upload/uploader.py

```diff
@@ -400,7 +400,8 @@ def __init__(self, config_file_path: str):
     def parse_file(self):
         config_file_data = self.get_file_data()
         custom_packs_paths = self.get_custom_packs_paths(config_file_data)
-        return custom_packs_paths
+        csv_custom_packs_paths = ",".join(path for path in custom_packs_paths)
+        return csv_custom_packs_paths
```

Comment:
I prefer to do it in `get_custom_packs_paths` itself instead of return it as list from this method and convert to CSV here